content/coronavirus_landing_page.yml

```diff
@@ -62,6 +62,28 @@ content:
         url: https://www.nidirect.gov.uk/campaigns/coronavirus-covid-19
   statistics_section:
     header: UK statistics
+    tests:
+      title: People tested positive
+      daily_heading: Daily
+      weekly_heading: Last 7 Days
+      guidance_link:
+        label: All case data
+        url: https://coronavirus.data.gov.uk/details/cases
+    admissions:
+      title: Patients admitted to hospital
+      daily_heading: Daily
+      weekly_heading: Last 7 Days
+      guidance_link:
+        label: All healthcare data
+        url: https://coronavirus.data.gov.uk/details/healthcare
+    vaccinations:
+      title: People vaccinated, percent of population aged 16+
```

Comment:
Change to:
> People vaccinated, and percent of population aged 12 and over vaccinated

Reason: data includes those aged 12 and over now (rather than 16+) and is clearer than saying 12+